### PR TITLE
Beautify dependencies declaration with ng-annotate

### PR DESCRIPTION
**Pull request by igrep**
*Review after you recover from your illness. :ambulance: Take care.*

I found a nice (maybe vital for every angular project) npm package named [ng-annotate](https://github.com/olov/ng-annotate) and its [grunt task](https://github.com/mzgol/grunt-ng-annotate).

So I have successfully eliminated the terribly long controller / service declarations, which makes it tough to add or remove dependencies.
Feel free to comment if you find any misconfiguration of Gruntfile.js, or if you don't like my style.
I'm new to Grunt.

### CODE REVIEW

**Comment by yoichiro** [comment]
Can one of the admins verify this patch?

**Comment by igrep** [comment]
@yoichiro ping.

**Comment by yoichiro** [comment]
test this please

File: app/scripts/window/controllers/procedures_functions_panel_controller.js [hunk 1]
```diff
@@ -1,4 +1,18 @@
-chromeMyAdmin.controller("ProceduresFunctionsPanelController", ["$scope", "mySQLClientService", "modeService", "targetObjectService", "UIConstants", "Modes", "Events", "mySQLQueryService", "Templates", "routineSelectionService", "$q", "anyQueryExecuteService", function($scope, mySQLClientService, modeService, targetObjectService, UIConstants, Modes, Events, mySQLQueryService, Templates, routineSelectionService, $q, anyQueryExecuteService) {
+chromeMyAdmin.controller("ProceduresFunctionsPanelController"
+,
+function($scope,
```
Comment (yoichiro) [inline app/scripts/window/controllers/procedures_functions_panel_controller.js]: Could you modify this code as like other code style?

Comment (igrep) [inline app/scripts/window/controllers/procedures_functions_panel_controller.js]: :+1:

**Pull request merged**